helm/kube-prometheus-stack/values.yaml

```diff
@@ -330,6 +330,9 @@ kube-prometheus-stack:
       enabled: true
     verticalPodAutoscaler:
       enabled: true
+      minAllowed:
```

Comment:
Can you add a comment to explain why we set this?

Reply:
Done